### PR TITLE
Fixing filter ordering

### PR DESCRIPTION
**Pull request by aasimkhan30**
image

This PR fixes https://github.com/microsoft/vscode-mssql/issues/18157

### CODE REVIEW

File: Packages.props [hunk 1]
```diff
@@ -77,6 +77,7 @@
     <PackageReference Update="System.Text.Encoding.CodePages" Version="8.0.0" />
     <PackageReference Update="System.Text.Encodings.Web" Version="8.0.0" />
     <PackageReference Update="System.Text.Json " Version="8.0.4" />
+    <PackageReference Update="System.Drawing.Common" Version="5.0.3" />
```
Comment (caohai) [inline Packages.props:80]: How is `System.Drawing.Common` currently used? I see the build failed with CG error related to it.

File: src/Microsoft.SqlTools.Migration/Microsoft.SqlTools.Migration.csproj [hunk 2]
```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="Microsoft.SqlServer.Migration.Tde" />
     <PackageReference Include="Microsoft.Data.SqlClient" />
 		<PackageReference Include="Microsoft.SqlServer.SqlTargetProvisioning" />
+    <PackageReference Include="Microsoft.Drawing.Common" />
```
Comment (caohai) [inline src/Microsoft.SqlTools.Migration/Microsoft.SqlTools.Migration.csproj:32]: `Microsoft.Drawing.Common` or `System.Drawing.Common`?

File: Packages.props (hunk 1 above)
Comment (aasimkhan30) [reply Packages.props:80]: I think this package is indirectly referenced by migration nugets. I am force updating it to unblock build pipelines.

Comment (caohai) [reply Packages.props:80]: makes sense

**Review by caohai — APPROVED** [review]

File: src/Microsoft.SqlTools.SqlCore/ObjectExplorer/SmoModel/SmoTreeNodes.cs
Comment (aasimkhan30) [inline src/Microsoft.SqlTools.SqlCore/ObjectExplorer/SmoModel/SmoTreeNodes.cs:1]: Autogen file please ignore.

**Comment by Benjin** [comment]
Are all the package ref updates part of the reordering changes, or unrelated?

**Review by Benjin — APPROVED** [review]

**Comment by aasimkhan30** [comment]
@Benjin created a separate PR for those. Reverting package updates from this PR

**Pull request merged**